test/runtests.jl

```diff
@@ -75,6 +75,20 @@ struct CubeRooter end
     @test collect(x) == [1, 2, 4]
     @test collect(x) isa Vector{Int}
 
+    x = [1, 2, missing, 4]
+    y = [1, 2, 3, missing]
+    z = [missing, missing, 3, 4]
```

Comment:
```suggestion
    z = [missing, missing, 3.1, 4.5]
```
And then test the different eltypes.